doc/snippets/Microsoft.Data.SqlClient/SqlConnection.xml

```diff
@@ -472,6 +472,23 @@ The connection string contains <see langword="Context Connection=true" />.
             <value>The list of trusted master key paths for the column encryption.</value>
             <remarks>To be added.</remarks>
         </ColumnEncryptionTrustedMasterKeyPaths>
+		<CommandTimeout>
+			<summary>
+				Gets the default wait time (in seconds) before terminating the attempt to execute a command and generating an error. The default is 30 seconds.
+			</summary>
+			<value>
+				The time in seconds to wait for the command to execute. The default is 30 seconds.
+			</value>
+            <remarks>
+            <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+ You can set default wait time by using the `Command Timeout` keyword in the connection string. A value of 0 indicates no limit (an attempt to execute a command will wait indefinitely).
```

Comment:
```suggestion
 You can set the default wait time by using the `Command Timeout` keyword in the connection string. A value of 0 indicates no limit (an attempt to execute a command will wait indefinitely).
```

Reply:
Can you also add the same warning found in SqlConnection.ConnectionTimeout to avoid a value of 0?

Reply:
Not sure I agree, sometimes you want a very long running command to complete.

Reply:
Typo fixed

Reply:
I also do not see a warning about 0 in the SqlCommand.CommandTiemout docs.

Reply:
> I also do not see a warning about 0 in the SqlCommand.CommandTiemout docs.

The warning is in SqlConnection, not SqlCommand. But never mind that, you're right about long running commands.

Can you also update the table of keywords in SqlConnection.ConnectionString?